### ISSUE TITLE
Groups are translated twice.

### ISSUE THREAD

**Issue by pogiaron** [evancz/elm-playground#3]
<pre>
import Playground exposing (..)

main =
  game view update (0,0)

view computer (x,y) =
  [ tile
  ]
  
tile =
  group 
  [ square yellow 40
  ]

update computer (x,y) =
  ( x + toX computer.keyboard
  , y + toY computer.keyboard
  )
</pre>

From the code above this svg will be generated:

<pre>
&lt;g transform=&quot;translate(571.5,377)&quot;&gt;
    &lt;rect width=&quot;40&quot; height=&quot;40&quot; fill=&quot;#edd400&quot; transform=&quot;translate(571.5,377) translate(-20,-20)&quot;&gt;&lt;/rect&gt;
&lt;/g&gt;
</pre>

So my square will not appear in the middle, rather in the left right corner, because the group and the rect element is also translated.

**Comment by evancz** [comment]
I have some changes that work with the following example:

```elm
import Playground exposing (..)


main =
  game view update (0,0)


view computer (x,y) =
  [ face
      |> move x y
      |> rotate (degrees (40 * x + 30 * y))
  , hello
      |> moveDown 240
      |> moveRight 200
      |> rotate 10
  ]


update computer (x,y) =
  ( x + toX computer.keyboard
  , y + toY computer.keyboard
  )


-- GROUPS


face =
  group
    [ circle lightYellow 200
    , eye
        |> moveUp 30
        |> moveLeft 60
    , eye
        |> moveUp 30
        |> moveRight 60
    , smile
        |> moveDown 70
    ]


eye =
  group
    [ circle white 40
    , circle black 10
        |> moveDown 10
    ]


smile =
  group
    [ oval black 200 100
    , rectangle lightYellow 200 80
        |> moveUp 12
    ]


hello =
  group
    [ words black "HELLO"
        |> scale 4
    , rectangle black 220 4
        |> moveDown 30
    ]
```

I tried to use a bunch of groups that had different kinds of transformations internally. The hope was that if the new system had some mistake, it would manifest when using a bunch of things together like this. (Like if the text was the wrong way around!)

**Comment by evancz** [comment]
Okay, the fixes should be available in 1.0.2. Thank you for the report, and please let me know if you run into anything else!

**Comment by evancz** [comment]
The fixed version is live on https://elm-lang.org/try as well, so you should be able to test things out there as well.
